shell/platform/embedder/BUILD.gn

```diff
@@ -27,8 +27,6 @@ template("embedder_source_set") {
       "embedder.cc",
       "embedder_engine.cc",
       "embedder_engine.h",
-      "embedder_external_texture_gl.cc",
```

Comment:
What happened to the 4 deleted files in here?

Reply:
Oops, good catch. I forgot to re-add them for the GL-enabled platforms. I'll sort that out :)